contracts/tasks/aero.js

```diff
@@ -0,0 +1,162 @@
+const { formatUnits } = require("ethers").utils;
+const { BigNumber } = require("ethers");
+const { getBlock } = require("./block");
+const { resolveAsset, resolveContract } = require("../utils/resolvers");
+const { getSigner } = require("../utils/signers");
+const { base } = require("../utils/addresses");
+
+const snapAero = async ({ block }) => {
+  const signer = await getSigner();
+  const blockTag = await getBlock(block);
+
+  const weth = await resolveAsset("WETH");
+  const vault = await resolveContract("OETHBaseVaultProxy", "IVault");
+  const oeth = await resolveContract("OETHBaseProxy", "OETHBase");
+  const pool = await resolveContract(base.aerodromeOETHbWETHClPool, "ICLPool");
```

Comment:
nit: could be read from the strategy contract: 
https://github.com/OriginProtocol/origin-dollar/blob/master/contracts/contracts/strategies/aerodrome/AerodromeAMOStrategy.sol#L75

Reply:
could, but I'll leave it from addresses.js